inspirehep/modules/theme/views.py

```diff
@@ -658,6 +659,19 @@ def linkedaccounts():
 
 
 #
+# Record preview
+#
+
+@blueprint.route('/record/preview', methods=['POST'])
```

Comment:
The thing here is that the `recid` parameter would be unused if passed, since there is no fetching being done based on that `recid`. Instead of that, the whole record is passed from the editor.

Also we might want to use this endpoint for non-existing records, doing a preview from a Holding Pen entry (which does not have recid yet) - although this might break some assumptions in our current detailed view.

> HTTP method should semantically be GET

Indeed, but since the whole JSON needs to be passed in the request from the editor, adding the whole JSON in a URL query param might get truncated.

(just for reference, this is [the line](https://github.com/inspirehep/record-editor/pull/109/files#diff-bef73574144e5725cded1a7ef2bed4c9R48) where I am using this endpoint at the moment)